data/docs-nav.ts

```diff
@@ -48,6 +48,7 @@ const DocsNav: NavItem[] = [
     path: "/docs/config",
     children: [
       { title: "Apps", path: "/docs/config/apps" },
+      { title: "Models", path: "/docs/config/models" },
```

Comment:
I think models should come before apps maybe?

Reply:
Should it?  I was trying to think in terms of hierarchy -- an App can have more than one Model, but a Model can only be associated with one App, right?  I was also thinking in terms of the order they'll appear in the UI navbar.  Happy to make the switch, I'd just like to understand more!

Reply:
This is a good taxonomy question!  Both being "first" is possible because Apps and and Models don't meet up logically until Sources.

```
App    ↘
         Source -> Property -> Group... 
Model  ↗ 
```

* You could have one app (datawarehouse) that has many models (customer, lead, events).  App -> Model
* You could have many apps that each contribute parts to many models (snowflake has both the "users" table for customers and leads; redshift has events that are used for both customers and leads to calculate LTV).   Model -> App

After writing this out, I think you have it right initially - let's continue to talk about Apps first because that's probably how data-engineers think - I've got a datawehouse full of tables, not a collection of models that happen to be in a datawarehouse.